.github/workflows/PR-3.4.yaml

```diff
@@ -2,9 +2,14 @@ name: PR:3.4
 
 on:
   pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
     branches:
       - 3.4
 
+concurrency:
+  group: ${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
```

Comment:
Observing the nest of this feature from CI repository. If I add it in CI repository and keep it clean here, canceling is working well.

- Changes in CI repo: https://github.com/asenyaev/ci-gha-workflow/commit/743ba5c81f364a3e42eb3dc7ce7bb868aafc4900
- Use these changes: https://github.com/asenyaev/opencv/commit/267fda11941e9c1eee7a6f5e607874aaec0f8f72
- Add new commit over the previous: https://github.com/asenyaev/opencv/commit/bea5fc2db4082e414e8bdcf1166da428a7479ceb
- The previous jobs are canceled: https://github.com/asenyaev/opencv/actions/runs/3295909485